### ISSUE TITLE
Photo base64 data?

### ISSUE THREAD

**Issue by bitwombat** [jhermsmeier/node-vcf#8]
Does node-vcf intentionally dispose of the base64 photo data?  I don't see it treated specially in the source.  If this is a bug, I'll drum up a testcase...

e.g.
```
PHOTO;ENCODING=BASE64;JPEG:/9j/4AAQSkZJRgABAQAAAQ
 CAgICAgQDAgICAgUEBAMEBgUGBgYFBgYGBwkIBgcJBwYGCAs
 2wBDAQICAgICAgUDAwUKBwYHCgoKCgoKCgoKCgoKCgoKCgoK
 goKCgoKCgoKCgoKCgr/wAARCABgAGADASIAAhEBAxEB/8QAH
```

**Comment by jhermsmeier** [comment]
Dispose as in "ignore"? Also, is this happening with the current version `0.2.2` or the potential `1.0.0`?
I just added a test case to `next`, and it seems to be working – note that the string values of properties on the `next` version need to be accessed via `prop.valueOf()` – which I haven't written any documentation for yet.

Example (`1.0.0`):
```js
// Keep in mind that this can be an array,
// if there are multiple `photo` properties
var photo = card.get( 'photo' )
var data = photo.valueOf()
// > '/9j/4AAQSkZJRgABAQAAAQCAgICAgQDAgICAgUE...'
photo.encoding
// > 'BASE64'
photo.type
// > 'jpeg'
photo.is('jpeg')
// > true
```

Test output:
```
node-vcf (next) $ mocha --ui tdd test/photo.js

  vCard
    Photo
vCard {
  version: '4.0',
  data:
   { version: [String: '4.0'],
     photo: { [String: '/9j/4AAQSkZJRgABAQAAAQCAgICAgQDAgICAgUEBAMEBgUGBgYFBgYGBwkIBgcJBwYGCAs2wBDAQICAgICAgUDAwUKBwYHCgoKCgoKCgoKCgoKCgoKCgoKgoKCgoKCgoKCgoKCgr/wAARCABgAGADASIAAhEBAxEB/8QAH'] encoding: 'BASE64', type: 'jpeg' } } }
      ✓ should be able to read base64 encoded photo


  1 passing (24ms)
```

**Comment by jhermsmeier** [comment]
That actually uncovered a missing return statement in `card.get()` – thanks :)

**Comment by bitwombat** [comment]
It was with master.  Trying on next, same thing.

Traced it down to a whitespace issue.  How .vcf comes out of an Android phone has this at the end:

``` 
(bunch of base64 stuff)
T9B1WO6F7o32hJbe4XyvLibc33WZW/hb/gVaQhZGUz/2Q==

END:VCARD
```

The (right?) [RFC](https://tools.ietf.org/html/rfc6350) doesn't really address it since it's not a line folding issue (it's CR/LF followed by CR/LF, not a space or tab), but competing libraries have [noted it](https://github.com/schmurfy/vcard_parser/issues/5) and it has [plagued other projects](https://bugzilla.mozilla.org/show_bug.cgi?id=674841).

Up to you if you want to deal with it in node-vcf.

**Comment by jhermsmeier** [comment]
Okay, good to know about that, thank you!
I'm adding a test case now, and will see to fixing it, maybe even backporting it to `0.2.x`.

> Up to you if you want to deal with it in node-vcf.

Well, this is supposed to make people's encounters with vCards easier, so if it's stuff that's happening in real world data, I intend to deal with it :)

**Comment by jhermsmeier** [comment]
Patched and published in `0.2.3`, as well as `next` now - thanks again for reporting this!
